create_derivatives/tests.py

```diff
@@ -118,3 +142,42 @@ def test_run(self, mock_get_object, mock_init):
     def tearDown(self):
         for f in Path(settings.SRC_DIR).iterdir():
             f.unlink()
+
+
+class AWSUploadTestCase(TestCase):
+
+    def setUp(self):
+        """Creates a temp directory and adds an uncompressed bag fixture to the temp directory and database"""
+        path = Path(settings.TMP_DIR)
+        if not path.exists():
+            path.mkdir(parents=True)
+        bag = "3aai9usY3AZzCSFkB3RSQ9"
+        fixture_directory = "aws_upload_bag"
+        bag_path = Path(settings.TMP_DIR, bag)
+        if not Path(bag_path).exists():
+            shutil.copytree(Path("create_derivatives", "fixtures", fixture_directory, bag), bag_path)
+            Bag.objects.create(
+                bag_identifier="sdfjldskj",
+                bag_path=bag_path,
+                origin="digitization",
+                as_data="sdjfkldsjf",
+                dimes_identifier=bag,
+                process_status=Bag.MANIFESTS_CREATED)
```

Comment:
Would prefer if this were done via a fixture rather than manually creating objects. See the `BagPreparerTestCase`.

Reply:
@helrond I'm seeing the structure of `created.json` and I generally follow what's going on in it. Where I'm struggling is how those bag model objects are created. Does that happen automatically?

Reply:
Check the [documentation](https://docs.djangoproject.com/en/3.2/topics/testing/tools/#fixture-loading)!